cpp/test/grpc/wait_callback.cpp

```diff
@@ -171,7 +171,7 @@ namespace wait_callback_tests
         UT_AssertIsTrue(wasInvoked);
     }
 
-    static void Initialize()
+    void Initialize()
```

Comment:
These functions don't need to be linked with outside of this compilation unit, hence the `static`.

Reply:
Anonymous namespace is a better option in that case, but here it is anyway inside of a test namespace, so did not bother adding that.

Reply:
Anyway, when we switch to a Boost auto test, they'll no longer be static.